en/docs/learn/design-api/advanced-topics/control-api-visibility-and-subscription-availability-in-developer-portal.md

```diff
@@ -123,6 +123,48 @@ When using the REST API directly, the subscription availability options are avai
     </tr>
     </tbody>
     </table>
+    
+    
+## Control API Document Visibility in Developer Portal
+
+By default, any document associated with an API has the same visibility level of the API. That is, if the API is public, 
+its documentation is also visible to all users (registered and anonymous).
```

Comment:
unnecessary spaces before this line. Please check the same in the entire document.

Reply:
fixed in 5ac01b7